src/Datasource/ConnectionInterface.php

```diff
@@ -18,6 +18,7 @@
  * This interface defines the methods you can depend on in
  * a connection.
  *
+ * @method object getDriver() Gets the driver instance.
```

Comment:
We'll need a similar annotation in `DriverInterface` too for the `newTableSchema()` method.